operator/controllers/csiscaleoperator_controller.go

```diff
@@ -1992,15 +1847,11 @@ func (r *CSIScaleOperatorReconciler) handleSpectrumScaleConnectors(instance *csi
 					continue
 				}
 				if id != cluster.Id {
-					message := fmt.Sprintf("The cluster ID %v in driver manifest does not match with the cluster ID %v obtained from cluster", cluster.Id, id)
+					message := fmt.Sprintf("The cluster ID %v in Spectrum Scale cluster properties does not match with the cluster ID %v obtained from cluster. Please check the Spec.Clusters in the resource %s/%s", cluster.Id, id, instance.Kind, instance.Name)
```

Comment:
Please change as above: "cluster ID %v in Spectrum Scale CSI configuration does not match ..."